src/@types/xmldom/index.d.ts

```diff
@@ -0,0 +1,14 @@
+// Type definitions for xmldom 0.1.22
+// Project: https://github.com/xmldom/xmldom
+// Definitions by: Qubo <https://github.com/tkqubo>
+//                 Karfau <https://github.com/karfau>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+/// <reference lib="dom" />
+
+import * as xmldom from "xmldom";
```

Comment:
do you plan to submit this bug (security issue?) to the `DefinitelyTyped` repo?

Reply:
FYI https://github.com/DefinitelyTyped/DefinitelyTyped/discussions/65454